stable/cosbench/templates/controller-deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "cosbench-controller.fullname" . }}
+  labels:
+    app: {{ template "cosbench.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    role: controller
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "cosbench.name" . }}
+      release: {{ .Release.Name }}
+      role: controller
+  template:
+    metadata:
+      labels:
+        app: {{ template "cosbench.name" . }}
+        release: {{ .Release.Name }}
+        role: controller
+    spec:
+      containers:
```

Comment:
One thing I missed in the previous review was RBAC: see https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md for howto
If there are no interactions with the actual k8s api, just add a service account, if there is interaction with api - add roles and rolebindings too.

Reply:
Good point, I just added `serviceAccounts`.